_posts/2018-02-02-workflow-process-manager.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Using a process manager with the workflow component
+author: Georgos Landin Xanthopoulos
+date: '2018-02-02 15:06:47 +0200'
+header:
+  image: "images/posts/little-man-hacker.jpg"
+  teaser: images/posts/little-man-hacker.jpg
+  caption: "Photo credit: [**The Preiser Project**](https://www.flickr.com/photos/thepreiserproject/)"
+categories:
+- Symfony
+---
+
+A workflow is very similar to a state machine, containing places and transitions, where places refer to "states".
+To read more about configuring and setting up your own workflow, please see the Symfony documentation:[here](https://symfony.com/doc/current/components/workflow.html)
```

Comment:
Is this interesting for this topic?

Reply:
I think it could be interesting for some readers, I also like having a small introduction, before continuing on to our main issue